src/igvfd/schemas/changelogs/cell_line.md

```diff
@@ -1,5 +1,9 @@
 ## Changelog for *`cell_line.json`*
 
+### Minor changes since schema version 2
+
+* `part_of` property restricted to be linked only to `cell_line`.
```

Comment:
To keep the changelog consistent I think we can reword the sentence. We are trying to start each line with a present tense verb now, so maybe something like:
Restrict `part_of` property to link to `cell_line` [or `primary_cell`, `tissue`, etc.] only.